### DISCUSSION TITLE
Alerting: hard fail if the minimum interval setting is negative or not multiple of the scheduler interval

### DISCUSSION THREAD

**Discussion by papagian** [grafana/grafana#39709] [Alerting]
Currently, in that case, we fallback enforcing the minimum possible interval which is the scheduler interval (10 seconds).
Following [this suggestion](https://github.com/grafana/grafana/pull/35623/files#r710969950) this should change and we should stop the server from starting so that we alarm the users that this configuration is invalid.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
